cdisc_rules_engine/services/data_readers/dataset_json_reader.py

```diff
@@ -0,0 +1,58 @@
+import pandas as pd
+import json
+
+from cdisc_rules_engine.interfaces import (
+    DataReaderInterface,
+)
+
+# Testing only
+# test_file_path = "D:\\CDISC_CORE_Engine\\Python_work_Jozef\\lb.json"
+
+
+class DatasetJSONReader(DataReaderInterface):
+    def from_file(self, file_path):
+        # print("dataset_json_reader: starting reading_from_file = ", file_path)
+        f = open(file_path)
+        # returns JSON object as
+        # a dictionary
+        data = json.load(f)
+        # print("data = ", data)
+        # check for "clinicalData"
+        start_data_string = "clinicalData"
+        if 'clinicalData' in data:
```

Comment:
is this check necessary if we are not changing start_data_string?

Reply:
The reason is that in Dataset-XML most datasets have "clinicalData" as the object name, but the trial design datasets (like TS, TA, TI, TE, etc.) have "referenceData" as the object name. We can however not rely on the dataset name for that (there are exceptions to the rule that trial design datasets start with a "T").
It can however be that the are some lines are not necessary (are from early) attempts, so I will check again.